src/Randomizer.SMZ3.Tracking/tracker.json

```diff
@@ -1498,14 +1520,17 @@
       "Picked up more {0}",
       "Even more {0}",
       "Cool, more {0}",
-      "Your {0} collection is getting somehwere.",
+      "Your {0} collection is getting somewhere.",
```

Comment:
I believe `{0}` is the plural form, but this should be singular. Need to check.

Reply:
Yup, you're correct. I had forgotten, but I remember catching that on a stream. I added a singular parameter and updated a couple of these to use it.